app/private_publisher.py

```diff
@@ -40,8 +40,8 @@ def decrypt(token, secret):
             message = f.decrypt(token.encode("utf-8"))
         return message.decode("utf-8")
 
-    def publish_message(self, message, content_type=None, headers=None, secret=None):
+    def publish(self, message, content_type=None, headers=None, secret=None):
```

Comment:
If _publish_message_ for this class is now renamed _publish_, no need to call super(). Base class _publish_message_ can be referenced from self.